pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["flit_core >=3.2,<4", "setuptools"]
+build-backend = "flit_core.buildapi"
+src-layout = "laser_encoders"
+
+[project]
+name = "laser_encoders"
+version = "0.0.1"
+authors = [{name = "Facebook AI Research"}]
+description = "LASER  Language-Agnostic SEntence Representations is a toolkit to calculate multilingual sentence embeddings and to use them for document classification, bitext filtering and mining"
+readme = "README.md"
+requires-python = ">=3.8"
+
+dependencies = [
+'sacremoses==0.0.53',
+'sentencepiece==0.1.99',
+]
+
+classifiers=[
+    "License :: OSI Approved :: MIT License",
```

Comment:
The license is actually BSD: https://github.com/facebookresearch/LASER/blob/main/LICENSE